src/controllers/ArticleController.js

```diff
@@ -11,15 +11,14 @@ export default class ArticleController {
    * @returns {object} Object representing the response returned
    */
   static async saveArticle(req, res) {
-    const image = await helpers.upload(req);
     const { tagList } = req.body;
     const newArticle = await Article.create({
       userId: req.user.id || 0,
       slug: helpers.generator.slug(req.body.title),
       title: req.body.title.trim(),
       description: req.body.description.trim(),
       body: req.body.body.trim(),
-      coverUrl: image.image.original,
+      coverUrl: req.body.coverUrl || '/templates/images/image.png',
```

Comment:
is the default `coverURL` value supposed to help with those that don't want to upload images?

Reply:
YES

Reply:
i feel like there should be a more elaborate way of doing this, we don't have to add a dummy image